docs/topic-guides/hp-tuning-det/index.txt

```diff
@@ -15,35 +15,33 @@ parallelization of these methods.
 
 Adaptive Search
 ---------------
-Our default recommended search method is adaptive search, an early-stopping
-based technique that speeds up traditional techniques like random search by
-periodically abandoning low-performing hyperparameter configurations in a
-principled fashion. Adaptive search builds on the two prototypical adaptive
+Our default recommended search method is
+(`Adaptive (ASHA) <https://arxiv.org/pdf/1810.05934.pdf>`_), a state-of-the-art
```

Comment:
Extra set of parens here. Also, do we want to link to the paper directly? Seems more useful to keep users within our docs.